src/opendr/perception/object_detection_2d/centernet/README.md

```diff
@@ -0,0 +1,17 @@
+OpenDR 2D Object Detection - CenterNet
+======
+
+This folder contains the OpenDR Learner class for CenterNet for 2D object detection.
+
+Sources
+------
+Large parts of the Learner code are taken from 
+[GluonCV's CenterNet implementation](https://www.github.com/dmlc/gluon-cv/tree/master/scripts/detection/center_net) with 
+modifications
+to make it compatible with OpenDR specifications.
```

Comment:
```suggestion
Large parts of the Learner code are taken from [GluonCV's CenterNet implementation](https://www.github.com/dmlc/gluon-cv/tree/master/scripts/detection/center_net) with modifications to make it compatible with OpenDR specifications.
```

Reply:
To avoid having to deal with formatting, we usually write one sentence per line in markdown files.